### PR TITLE
Documentation and Readme Update

### PR DESCRIPTION
**Pull request by vvolkl**
BEGINRELEASENOTES
- add  diagram showing association between types
- improve whitespace in edm4hep.yaml
- update Readme

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
Close and open to trigger CI

**Comment by vvolkl** [comment]
Find [here](https://github.com/vvolkl/EDM4HEP/tree/update-readme) the rendered Readme.

**Comment by andresailer** [comment]
> fix MCRecoTrackerAssociation (forgotten sim type)

This was there, but was dropped in one commit and then added again, wasn't it?

**Comment by vvolkl** [comment]
Hi @andresailer, indeed, good catch! I thought I had overlooked this in #23 
Should be ok now.

File: README.md [hunk 1]
```diff
@@ -4,15 +4,63 @@
...
 
 
-## Work in Progress ...
 
-This project has just started and is still very experiemental.
 
 
 ## Contributing
 
+Contributions and bug reports are welcomed!
```
[the remark this answers is not in the document]
Comment (andresailer) [reply README.md]: I think this `d` is back?

[the remark this answers is not in the document]
Comment (vvolkl) [reply README.md]: Ah yes, I had not pulled this commit from github sorry about that.

**Pull request merged**